jekyll/_cci2/api-developers-guide.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: classic-docs
+title: "CircleCI API Developer's Guide"
+short-title: "Developer's Guide"
+description: "API cookbook for internal and external CircleCI developers"
+categories: [getting-started]
+order: 1
+---
+
+This *API Developer's Guide* was written to assist developers in quickly and easily making API calls to CircleCI services to return detailed information about users, pipelines, projects and workflows.
+
+* TOC
+{:toc}
+
+# API Overview
+
+The CircleCI platform provides a powerful API that enables users to retrieve detailed information about users, jobs, workflows and pipelines.
+
+**Note:** Whenever you see a term prefixed with a colon (:), please note that this value must be replaced by a user-input value. For example `:token` in the above call must be replaced by the token string generated in your account:
+
+curl https://circleci.com/api/v1.1/me?circle-token=dkanmbowk34iajdkl3iakdthisisarandomtoken
```

Comment:
Replace with example using v2 api?

Reply:
esp here because there doesn't seem to actually be a `:token` part to this request? Maybe that's a v2 thing?

Reply:
Fixed